style.css

```diff
@@ -382,3 +417,173 @@ body {
   color: white;
   cursor: pointer;
 }
+
+.cont-menu-list {
+  display: none;
+}
+
+@media (min-width: 600px) {
+  .cont-menu-list {
+    list-style: none;
+    display: flex;
+  }
+
+  .menu__cont i {
+    display: none;
+  }
+
+  .link-menu {
+    text-decoration: none;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: black;
+  }
+
+  .menu-list > * {
+    margin-right: 1rem;
+  }
+
+  .intro {
+    width: 80%;
+    margin: 7rem auto 0 auto;
+  }
+
+  .container__card {
+    width: 75%;
+  }
+
+  .cont-inp-form {
+    width: 70%;
+    margin: 0 auto;
+  }
+
+  .item-field {
+    width: 90%;
+    margin-right: auto;
+    margin-left: auto;
+  }
+
+  .cont-btn-form {
+    align-self: center;
+  }
+}
+
+@media (min-width: 900px) {
+  .intro {
+    width: 75%;
+    margin: 10rem auto 2rem auto;
+  }
+
+  .card__img--cont {
+    flex-direction: row;
+  }
+
+  .prjct-info-wrapper {
+    width: 50%;
+    margin-left: 1rem;
+  }
+
+  .prjct-info-wrapper > *:not(:last-child) {
+    margin-bottom: 1rem;
+  }
+
+  .img-wrapper {
+    width: 50%;
+  }
+
+  .container__card {
+    width: 85%;
+  }
+
+  .card-2,
+  .card-4 {
+    flex-direction: row-reverse;
+  }
+
+  .res-skil-cont {
+    display: flex;
+    flex-direction: row;
+    height: 500px;
+    align-items: center;
+    width: 80%;
+    margin: 0 auto;
+  }
+
+  .list__skills--cont {
+    margin-bottom: 0;
+  }
+
+  .intro__resume {
+    width: 40%;
+    align-self: flex-start;
+    margin: 0;
+  }
+
+  .skills__cont {
+    width: 30%;
+    margin: 0 1rem 0 0;
+  }
+
+  .lang-cont-list {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .cont__skill {
+    margin-bottom: 0;
+  }
+
+  .lang-info {
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    margin-right: 1rem;
+    margin-bottom: 0;
+    width: auto;
+  }
+
+  .cont-inp-form {
+    width: 60%;
+    margin: 0 auto;
+  }
+}
+
+@media (min-width: 1200px) {
```

Comment:
Please write the value for min-width property should in range of 992px - 1100px.